news/index.md

```diff
@@ -19,6 +17,19 @@ PGroonga requires Groonga 11.0.6 since this version.
     Added `index_flags_mapping` option that can be used to customize
     index flags for each indexed target.
 
+  * [[`CREATE INDEX USING PGroonga`][create-index-using-pgroonga-custom-normalizer]]
+    Added support for `${table:INDEX_NAME}` substitution in `normalizers_mapping`
+    option.
+
+  * Added support for PostgreSQL 14.
+
+  * [[Ubuntu][ubuntu]] Added support for Ubuntu 21.04.
+
+### Fixes
+
+  * [[`pgroonga_highlight_html` function][highlight-html]]
+    Fixed a bug that a lexicon may not update when we recreate the lexicon.
```

Comment:
even when different options are specified

Reply:
wrong result may be returned when `pgroonga_highlight_html` calls with different options exist in a `SELECT`